problems/4.arrays.js

```diff
@@ -1,31 +1,58 @@
 /** Averages the first and last value of an array.  Return null if the array has fewer than 2 elements.
-* @param {number[]} arr - The input array
-* @returns {number} - The average of the first and last values
-*/
-function firstAndLastAverage() {
+ * @param {number[]} arr - The input array
+ * @returns {number} - The average of the first and last values
+ */
+function firstAndLastAverage(arr) {
+    let newArray = [];
```

Comment:
Too much work! 
First element is arr[0] , last is arr[arr.length - 1] Average would be those numbers added together and divided by 2.

Reply:
ok understood.